communication/slack-config/channels.yaml

```diff
@@ -321,6 +321,7 @@ channels:
   - name: litmus-dev
   - name: lokomotive
     archived: true
+  - name: lula
```

Comment:
there is already a lula in slack and we channels / IDs share a namespace >_< maybe lula-dev?